Flow.Launcher/ViewModel/SettingWindowViewModel.cs

```diff
@@ -61,6 +61,18 @@ public bool AutoUpdates
             }
         }
 
+        public bool AutoHideScrollBar
+        {
+            get 
+            {
+                return Settings.AutoHideScrollBar;
+            }
+            set
+            {
+                Settings.AutoHideScrollBar = value;
+            }
+        }
+
```

Comment:
It seems that you directly link the Binding to `Settings.AutoHideScrollBar`, why is this property needed?

Reply:
property is for the settings window

Reply:
because we need to save the state

Reply:
I don't think you need this one. WPF will directly modify Settings.AutoHideScrollBar property, which already save the state.

Reply:
👍 yes you are right, forgot about that. will make the change

Reply:
@taooceros done, please take another look

Reply:
@taooceros done, please take another look